.github/workflows/lint.yml

```diff
@@ -70,6 +70,7 @@ jobs:
           - "propagator-ot-trace"
           - "resource-detector-container"
           - "sdk-extension-aws"
+          - "util-http"
```

Comment:
Rename to opentelemetry-util-http to match the package name?

Reply:
Not a huge deal. We use shorter terms like "distro" as well for other packages.